### ISSUE TITLE
Enhancement: BG Music stays playing if music is muted.

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#2878]
## What is your suggestion, and why should it be implemented?
Pretty simple, if all audio tracks that were added to the chart are muted, than the BG music should keep playing in the background if spacebar is hit to play the song.

**Comment by Hundrec** [comment]
This suggestion sounds intuitive, but I'm having trouble thinking of cases where someone would want this to happen.

If I've muted all the tracks in the chart, then I might be seeing how the chart's note patterns flow.
In this case, I would rather have Artistic Expression muted so I can focus on the rhythms of the chart rather than the BGM. 

Of course, the theme could easily be toggled from the context menu, but it seems like an unfavorable compromise to have to turn off the theme globally just to focus on the chart.

**Comment by Average-FNF-Modder** [comment]
> This suggestion sounds intuitive, but I'm having trouble thinking of cases where someone would want this to happen.
> 
> If I've muted all the tracks in the chart, then I might be seeing how the chart's note patterns flow. In this case, I would rather have Artistic Expression muted so I can focus on the rhythms of the chart rather than the BGM.
> 
> Of course, the theme could easily be toggled from the context menu, but it seems like an unfavorable compromise to have to turn off the theme globally just to focus on the chart.

I just don't think it would really bother me at all to have it in the background. Maybe it would bother some people, so it could be an option or something to have my idea on or not.

**Comment by EliteMasterEric** [comment]
I don't think this makes sense since it would mess you up with the metronome and hitsounds.
